bm_provision.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
I think we don't need this script. I've updated this script as openstack_commands/node_manage.sh

Reply:
Ah, right! Removed